### ISSUE TITLE
MIT discussion about network/space and how to raise issues with them.

### ISSUE THREAD

**Issue by msdisme** [CCI-MOC/ops-issues#194]
- [x] identify exact current list of dependencies on mit

**Comment by naved001** [comment]
There are 4 networks we depend on from MIT.

1. VLAN 3801 (128.31.20.0/22). We used this for engage1. Our brocade switches in BU cages 2 and 4 connect to the CSAIL switch to access this network.
2. VLAN 10 (128.52.60.0/22). We used this for infrastructure in engage1 but we have to use CSAIL's DHCP since they like to control this network, which makes this less desirable. Our same brocade switches that carry VLAN 3801 also carries this. 
3. VLAN 3802 (128.31.24.0/22). For Kaizen openstack floating IPs. We don't have a direct connection to CSAIL switch, but instead it's via NEU->MeetMe->MIT->CSAIL.
4. VLAN 3803 (128.31.28.0/24). This is used for MaaS services and hosts, and some infrastructure VMs. Our Cisco switch in Kumo connects to a BU switch which carries this VLAN in addition to BU's public network (VLAN 105, 192.12.185.0/24).

We can contact help@csail.mit.edu for all our CSAIL networks. 

Re: rack space:

We have no servers racked in MIT space. The cluster we call enage1 at MOC is racked in BU cages 2 and 4 (Row 4 Pod A). We have 4 brocade switches racked in those cages and they are part of a brocade fabric where the rest of the brocade switches (5) are racked in MIT cages. We should figure who gets dibs on those switches. The brocade switches that connect to CSAIL are in our cages so that's good.
